### PR TITLE
File upload to existing release

### PR DESCRIPTION
**Pull request by khng**

### CODE REVIEW

File: out/release/release_finder_test.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	)
+	BeforeEach(func() {
+		pivnetClient = &releasefakes.FinderClient{}
+		finderErr = nil
+		productSlug = "special-product"
+	})
+
+	Describe("Find", func() {
+
+		Context("", func() {
```
Comment (pivotal-ivan-wang) [inline out/release/release_finder_test.go:29]: is this a placeholder for additional tests?

File: README.md [hunk 2]
```diff
@@ -217,6 +217,13 @@ for more details on the structure of the metadata file.
 
   If `true`, forces a re-upload of releases of releases and versions that are already present on Tanzu Network.
 
+* `files_only`: *Optional boolean.*
```
Comment (pivotal-ivan-wang) [inline README.md:220]: Is this boolean used? We ignored it when testing on our pipelines and looking at the logic it seems we only use ExistingRelease from the metadata.

File: acceptance/out_test.go [hunk 3]
```diff
@@ -142,159 +144,295 @@ var _ = Describe("Out", func() {
...
 			})
 
-			It("with 'override' true, it succeeds", func() {
-				outRequest.Params.Override = true
+			It("succeeds", func() {
+				release, err := pivnetClient.GetRelease(productSlug, version)
+				Expect(err).NotTo(HaveOccurred())
+				releaseID := release.ID
+
+				outRequest.Params.FilesOnly = true
```
Comment (pivotal-ivan-wang) [inline acceptance/out_test.go:416]: Is this line needed?

File: out/release/release_finder_test.go (hunk 1 above)
Comment (rheaton) [reply out/release/release_finder_test.go:29]: It was a reminder for potential errors, but isn't needed. Removing.

File: README.md (hunk 2 above)
Comment (rheaton) [reply README.md:220]: Excellent question -- i'll run the tests without it and remove it if it's not necessary, and update the docs :)

**Pull request merged**